docs/reference/offline-stores/remote.md

```diff
@@ -0,0 +1,45 @@
+# Remote Offline Store
+
+## Description
+
+The Python offline server is an Apache Arrow Flight Server.
```

Comment:
worth to mention that it uses a grpc communication protocol to exchange data?

Reply:
updated.